modules/machine-user-provisioned-limitations.adoc

```diff
@@ -1,13 +1,17 @@
 // Module included in the following assemblies:
 //
-// * machine_management/applying_autoscaling.adoc
```

Comment:
This module was not referenced in the applying_autoscaling.adoc assembly (and does not seem to have ever been). The note covering similar material was [revamped in late 2019](https://github.com/openshift/openshift-docs/commit/fd0ab1e1875869e2f38189df8fef6baddfbabbf4#diff-1b7bfe0e4a765b714945f3db30721c9a6fb3d1b5a6cd11d76937701aecf85f5e). Removing this line reference.